packages/client/src/generation/TSClient/PrismaClient.ts

```diff
@@ -544,6 +544,21 @@ export type TransactionClient = Omit<Prisma.DefaultPrismaClient, runtime.ITXClie
              Read more in our [docs](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/logging#the-log-option).
           `),
       )
+      .add(
+        ts
+          .property(
+            'transactionOptions',
+            ts
+              .objectType()
+              .add(ts.property('maxWait', ts.numberType).optional())
+              .add(ts.property('timeout', ts.numberType).optional()),
```

Comment:
This type does not include isolation level, where it runtime, the property exists and affects the execution. You can see how to do add isolation level property [here](https://github.com/prisma/prisma/blob/d212cd87d0e3586fde996f2c479226a3d53b22e1/packages/client/src/generation/TSClient/PrismaClient.ts#L174)